model/src/main/kotlin/CuratedPackage.kt

```diff
@@ -41,4 +41,18 @@ data class CuratedPackage(
      * A comparison function to sort packages by their identifier.
      */
     override fun compareTo(other: CuratedPackage) = pkg.id.compareTo(other.pkg.id)
+
```

Comment:
Commit message:

> The implementation of policies enforcing rules on package meta data would ideally consume not curated packages.

I don't think this is generally the case. Both makes sense, working on curated or non-curated metadata, depending on what you want to check. I'd just remove this sentence.